.github/workflows/check-pr.yml

```diff
@@ -0,0 +1,38 @@
+name: check-pr
+on:
+  pull_request:
+    types: [opened, reopened, labeled, unlabeled, synchronize]
+  pull_request_review:
+    types: [submitted, dismissed]
+jobs:
+  check-labels:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: RIOT-OS/check-labels-action@v1.0.0
+      with:
+        access_token: ${{ secrets.GITHUB_TOKEN }}
+        unset_labels: 'CI: needs squashing,State: waiting for other PR'
+        cond_labels: '(Process: needs >1 ACK,review.approvals>1),(Area: RDM,review.approvals>2)'
+  check-commits:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+      with:
+        # Use the SHA of the PR branch as-is, not the PR branch merged
+        # in master (default behavior in GH actions)
+        # See https://github.com/actions/checkout#checkout-pull-request-head-commit-instead-of-merge-commit
+        ref: ${{ github.event.pull_request.head.sha }}
+        fetch-depth: 0
+    - name: Fetch base branch
+      run:
+        git fetch origin '${{ github.base_ref }}:${{ github.base_ref }}'
+    - name: Run checks
+      env:    # enforce deactivation ubuntu label check
+        TRAVIS_PULL_REQUEST:
+        CI_PULL_NR:
+      run: |
+        ./dist/tools/pr_check/pr_check.sh "${{ github.base_ref }}"
+        RES1=$?
+        ./dist/tools/commit-msg/check.sh "${{ github.base_ref }}"
+        RES2=$?
+        [ "${RES1}" -eq 0 ] && [ "${RES2}" -eq 0 ]
```

Comment:
You could use a matrix strategy with fail-fast set to false. If you rename pr_check.sh to check.sh, the items in the matrix would just be `["pr_check", "commit-msg"]`.

Reply:
Yep just was doing that :-).